contracts/Gateway.sol

```diff
@@ -101,43 +107,47 @@ contract Gateway is ProtocolVersioned, Owned {
         workers = _workers;
         bounty = _bounty;
         uuid = _uuid;
-
+        conversionRate = _conversionRate;
```

Comment:
Should we check here that the conversion rate is not `0`? If it is we will not be able to call the TokenConversionLib.